docker/Dockerfile

```diff
@@ -16,18 +16,26 @@
 #
 
 # Usage:
-#   1. use ./build/dist to make binary distributions of Kyuubi or download a release
-#   2. Untar it and run the docker command below
-#      docker build -f docker/Dockerfile -t yaooqinn/kyuubi:tagname .
+#   Run the docker command below
+#      docker build --build-arg MVN_ARG="-Pmirror-cn" -f docker/Dockerfile -t apache/kyuubi:tagname .
 #   Options:
 #     -f this docker file
 #     -t the target repo and tag name
 #     more options can be found with -h
-# TODO: REPALCE it with offical spark image iff Apache Spark community deploy
-# Or make one after kyuubi to be setup under ASF
 
-ARG SPARK_IMAGE=yaooqinn/spark:3.0.3
-FROM ${SPARK_IMAGE}
+FROM maven:3.6-jdk-8 as builder
+
+ARG MVN_ARG
+
+ADD . /workspace/kyuubi
+WORKDIR /workspace/kyuubi
+
+RUN ./build/dist ${MVN_ARG} && \
+    mv /workspace/kyuubi/dist /opt/kyuubi && \
+    rm -rf ~/.m2 && \
+    rm -rf /workspace/kyuubi
+
+FROM openjdk:8-jdk
```

Comment:
this line is useless, already use `FROM maven:3.6-jdk-8 as builder`
`FROM openjdk:8-jdk`

Reply:
The first one `FROM maven:3.6-jdk-8 as builder` provides build Kyuubi runtime(mvn/java), and the second one `FROM openjdk:8-jdk` is for Kyuubi server runtime(java only). In order to reduce the final docker image size, we removed MAVEN here.